src/sentry/workflow_engine/models/detector.py

```diff
@@ -105,3 +106,28 @@ def detector_handler(self) -> DetectorHandler | None:
     def get_audit_log_data(self) -> dict[str, Any]:
         # TODO: Create proper audit log data for the detector, group and conditions
         return {}
+
+    def get_option(
```

Comment:
this method should probably only live on `ErrorDetector` (or just make the query wherever it's being used)